Runtime/AvatarCreator/Scripts/UI/Buttons/AvatarListItem.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using ReadyPlayerMe.AvatarCreator;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+public class AvatarListItem : MonoBehaviour
+{
+    [SerializeField] private Button customizeAvatarButton;
+    [SerializeField] private Button selectAvatarButton;
+    [SerializeField] private Button deleteAvatarButton;
+    public RawImage avatarImage;
+    public UnityEvent onImageLoaded;
+
+    /// <summary>
+    /// Adds the listeners to the button's onClick event.
+    /// </summary>
+    /// <param name="customizeAvatar">A function to run when the customize button is clicked</param>
+    /// <param name="selectAvatar">A function to run when the select button is clicked</param>
+    /// <param name="deleteAvatar">A function to run when the delete button is clicked</param>
+    public void AddListener(Action customizeAvatar, Action selectAvatar, Action deleteAvatar)
```

Comment:
I actually have an idea for reducing some code duplication here. 
You can add some small data type (like a struct) to hold avatarID and an enum (eg actionType) which would have 3 states Customize, Select, Delete etc then you would only need to pass 1 function AND it becomes more extensible later if there would be some other type of action (not that this is likely). 
I also think that this could then potentially be merged with the LoadIcon function so you only need to calll 1 "setup" function. What do you think?

Reply:
I can test something